.devcontainer/Dockerfile

```diff
@@ -42,4 +42,8 @@ RUN sh -c "$(wget -O- https://github.com/deluan/zsh-in-docker/releases/download/
 # Terraform
 RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add - && \
     apt-add-repository "deb [arch=$(dpkg --print-architecture)] https://apt.releases.hashicorp.com $(lsb_release -cs) main" && \
-    apt-get -y install terraform
\ No newline at end of file
+    apt-get -y install terraform
+
+# Installs and executes the Rustup-init executable (https://forge.rust-lang.org/infra/other-installation-methods.html)
```

Comment:
This must also be installed on the containers (see [devops/docker](https://github.com/Azure/azure-osconfig/tree/main/devops/docker)), not just the dev containers. Also, any reason we cannot just use the `cargo` package?
https://packages.ubuntu.com/focal/cargo

Reply:
Dev containers will want all the other handy rust tools that are retrievable via the shell script. Cargo apt install will be used for all the devops containers, though. Added in a separate pr